include/readdy/api/SimulationLoop.h

```diff
@@ -100,8 +99,11 @@ class SimulationLoop {
               _integrator(kernel->actions().eulerBDIntegrator(timeStep).release()),
               _reactions(kernel->actions().gillespie(timeStep).release()),
               _forces(kernel->actions().calculateForces().release()),
-              _initNeighborList(kernel->actions().initNeighborList(kernel->context().calculateMaxCutoff()).release()),
-              _neighborList(kernel->actions().updateNeighborList().release()),
+              _initNeighborList(
```

Comment:
i'd always create the init neighbor list action or make this ternary comparison also for update and clear neighbor list. since it doesnt really cost anything, id probably do it in any case...

Reply:
it is now always created